packages/driver/cypress/integration/issues/5909+spec.js

```diff
@@ -0,0 +1,6 @@
+// https://github.com/cypress-io/cypress/issues/5909
```

Comment:
Let's add to [this e2e test](https://github.com/cypress-io/cypress/blob/develop/packages/server/test/e2e/4_controllers_spec.js#L23) instead of adding this driver test. It will give better coverage for this.

Reply:
Done